pkgs/development/python-modules/pylibftdi/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, httpserver
+, libftdi1
+, libusb1
+}:
+
+buildPythonPackage rec {
+  pname = "pylibftdi";
+  version = "0.18.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "17c5h4xz1grynbpffngjflk3dlw2g2zbhkwb7h5v4n9rjdv41l5x";
+  };
+
+  propagatedBuildInputs = [
+    httpserver
+    libftdi1
+    libusb1
+  ];
+
+  postPatch = ''
+    substituteInPlace pylibftdi/driver.py \
+      --replace "self._load_library('libusb')" "cdll.LoadLibrary('${libusb1.out}/lib/libusb1.so')" \
+      --replace "self._load_library('libftdi')" "cdll.LoadLibrary('${libftdi1.out}/lib/libftdi1.so')"
+  '';
+
```

to avoid the package being silently broken:
```suggestion
  '';

  pythonImportsCheck = [ "pylibftdi" ];
```

---

done